resources/styles/components/exercise-card.less

```diff
@@ -18,51 +14,55 @@
     background-color: @selected-exercise-background;
   }
 
-  .panel-heading {
-    position: absolute;
-    top: 0;
-    right: 0;
-    bottom: 0;
-    left: 0;
-    display: none;
-    background-color: fade(@tutor-neutral-lighter, 0%);
-    border: none;
-
-    .add-or-remove {
-      position: absolute;
-      top: 50%;
-      left: 50%;
-      height: 100px;
-      width: 100px;
-      margin-top: -50px;
-      margin-left: -50px;
-      background-color: fade(@tutor-white, 85%);
-      border: 1px solid @tutor-neutral-lighter;
-      border-radius: 100px;
-      color: @tutor-neutral;
-
-      &::before {
-        .fa-icon();
-        content: @fa-var-plus;
-        font-weight: 300;
-        text-align: center;
-        font-size: 50px;
-        line-height: 100px;
-        width: 100px;
-        position: relative;
+  &.is-selectable {
+    .panel-body {
+      position: relative;
+      &:hover {
+        .toggle-mask {
+          opacity: 0.85;
+          visibility: visible;
+        }
       }
+    }
 
-      &.active {
+    .toggle-mask {
+      cursor: pointer;
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      visibility: hidden;
+      z-index: 1;
+      opacity: 0;
+      transition: opacity .25s ease-in-out;
+      display: flex;
+      border: none;
+      justify-content: center;
+      align-items: center;
+
+      .message {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        font-size: 2.5rem;
+        font-weight: 700;
         &::before {
-          content: @fa-var-minus;
+          .fa-icon();
+          content: @fa-var-plus;
+          font-weight: 300;
+          text-align: center;
+          font-size: 50px;
+          border-radius: 60px;
+          width: 60px;
+          height: 60px;
+          padding-top: 5px;
         }
       }
     }
-  }
-
-  &:hover {
-    .panel-heading {
-      display: block;
+    &.is-selected .toggle-mask .icon::before {
```

Comment:
`&:hover` might be needed?  the "Exclude this Question" icon is the `@fa-var-plus` for some reason...also should `.icon` be `.message`?  i.e.

```less
    &.is-selected,
    &:hover {
      .toggle-mask .message::before {
        content: @fa-var-minus;
      }
    }
```